src/vm/threadstatics.h

```diff
@@ -42,7 +43,62 @@ struct ThreadLocalModule
     friend class CheckAsmOffsets; 
     friend struct ThreadLocalBlock;
 
+    // After these macros complete, they may have returned an interior pointer into a gc object. This pointer will have been cast to a byte pointer
+    // It is critically important that no GC is allowed to occur before this pointer is used.
+#define GET_DYNAMICENTRY_GCTHREADSTATICS_BASEPOINTER(pLoaderAllocator, dynamicClassInfoParam, pGCStatics) \
+    {\
```

Comment:
Please put on your list of work, making sure that these macros work correctly in the DAC. When I implemented collectible types, I recall the debugger support for statics being particularly difficult to get right.

Reply:
Added a new issue to track that.